README.md

```diff
@@ -192,17 +193,17 @@ Terminology
                   produce the finished article (built conda distributions)
 
 
-Updating spyder-feedstock
-=========================
+Updating spyder-base-feedstock
```

Comment:
@hmaarrfk, these changes may not be desirable. Since this is generated from the rerender, would it be better to make `spyder` the parent package and `spyder-base` the subpackage?

Reply:
Or do I manually revert some of these changes by the rerender?

Reply:
Don’t worry too much about these. They are about 90% correct. 

Maybe you can open an issue on our conda-forge.github.io to explain the kind of control you would want here so that we can better tune this